msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -468,6 +468,13 @@ internal IWebUI CreateWebAuthenticationDialog(UIParent parent, UIBehavior behavi
             var requestParams = CreateRequestParameters(authority, scopes, null, UserTokenCache);
             requestParams.ExtraQueryParameters = extraQueryParameters;
 
+#if iOS || ANDROID
+            if (!parent.CoreUIParent.UseEmbeddedWebview)

```

Comment:
>parent.CoreUIParent.UseEmbeddedWebview) [](start = 17, length = 39)

perhaps this should be available on all platforms? then the #if ios || android could be moved to the RedirectUriHelper validation?